libs/moloch-v3-context/src/utils/fetchHelpers.ts

```diff
@@ -1,303 +1,206 @@
 import { ReactSetter } from '@daohaus/utils';
-import { Keychain } from '@daohaus/keychain-utils';
+import { Keychain, ValidNetwork } from '@daohaus/keychain-utils';
 
 import {
   MolochV3Dao,
   findDao,
   findMember,
-  FindMemberQuery,
-  findProposal,
-  ListConnectedMemberProposalsQuery,
   listMembers,
-  ListMembersQuery,
   listProposals,
-  listProposalVotesByMember,
   Member_Filter,
   Member_OrderBy,
   Proposal_Filter,
   Proposal_OrderBy,
-  MolochV3Proposal,
 } from '@daohaus/moloch-v3-data';
-import deepEqual from 'deep-eql';
 import { Ordering, Paging } from '@daohaus/data-fetch-utils';
 
-export const loadDao = async ({
+export const fetchDao = async ({
   daoid,
   daochain,
-  setDao,
-  setDaoLoading,
-  shouldUpdate,
   graphApiKeys,
 }: {
   daoid: string;
   daochain: keyof Keychain;
-  setDao: ReactSetter<MolochV3Dao | undefined>;
-  setDaoLoading: ReactSetter<boolean>;
-  shouldUpdate: boolean;
   graphApiKeys?: Keychain;
 }) => {
   try {
-    setDaoLoading(true);
     const daoRes = await findDao({
       networkId: daochain,
       dao: daoid,
       includeTokens: true,
       graphApiKeys,
     });
 
-    if (daoRes?.data?.dao && shouldUpdate) {
-      setDao(daoRes.data.dao as MolochV3Dao);
+    if (daoRes?.data?.dao) {
+      return daoRes.data.dao as MolochV3Dao;
+    } else {
+      console.error('no dao found');
     }
   } catch (error) {
     console.error(error);
-    setDao(undefined);
-  } finally {
-    if (shouldUpdate) {
-      setDaoLoading(false);
-    }
   }
 };
 
-export const loadMember = async ({
+export const fetchMember = async ({
   daoid,
   daochain,
   address,
-  setMember,
-  setMemberLoading,
-  shouldUpdate,
   graphApiKeys,
 }: {
   daoid: string;
   daochain: keyof Keychain;
   address: string;
-  setMember: ReactSetter<FindMemberQuery['member'] | undefined>;
-  setMemberLoading: ReactSetter<boolean>;
-  shouldUpdate: boolean;
   graphApiKeys?: Keychain;
 }) => {
   try {
-    setMemberLoading(true);
-    const memberRes = await findMember({
+    const res = await findMember({
       networkId: daochain,
       dao: daoid,
       memberAddress: address.toLowerCase(),
       graphApiKeys,
     });
 
-    if (memberRes?.data?.member && shouldUpdate) {
-      setMember(memberRes.data.member);
-    } else if (shouldUpdate) {
-      setMember(undefined);
+    if (res?.data?.member) {
+      return res.data.member;
+    } else {
+      console.error('no member found');
```

Comment:
I'm wondering if we should add lifecycleFns so we can have some error handling. Could be a ticket for later

Reply:
Good idea - def makes sense for later